projects/storefrontapp-e2e-cypress/cypress/integration/regression/multisite-isolation/multisite-isolation.core-e2e-spec.ts

```diff
@@ -1,109 +1,50 @@
-import { clearAllStorage } from '../../../support/utils/clear-all-storage';
-import { user } from '../../../sample-data/checkout-flow';
-import { register } from '../../../helpers/auth-forms';
-import * as login from '../../../helpers/login';
-import * as alerts from '../../../helpers/global-message';
-
 import {
   ELECTRONICS_BASESITE,
-  ELECTRONICS_CURRENCY,
+  visitHomePage,
 } from '../../../helpers/checkout-flow';
-import { POWERTOOLS_BASESITE } from '../../../sample-data/b2b-checkout';
+import * as alerts from '../../../helpers/global-message';
+import * as isolation from '../../../helpers/multisite-isolation';
 
 export const ELECTRONICS_STANDALONE_BASESITE = 'electronics-standalone';
 export const POWERTOOLS_STANDALONE_BASESITE = 'powertools-standalone';
 
 describe('Multisite Isolation', () => {
   before(() => {
-    cy.window().then((win) => {
-      win.localStorage.clear();
-    });
-  });
-
-  beforeEach(() => {
-    clearAllStorage();
+    cy.window().then((win) => win.sessionStorage.clear());
   });
 
-  describe('Standalone authorization', () => {
+  describe('Non-Standalone authorization', () => {
```

Comment:
What is the difference between non-standalone and standalone authorization more specifically? Can we describe this in comments?